.env.example

```
@@ -55,6 +55,10 @@ NIGHTMARENET_CELERY_INCLUDE=nightmarenet.tasks
 # Device selection (auto, cpu, cuda)
 NIGHTMARENET_DEVICE=auto
 
+ feat/signed-json-compliance-export
```

dotenv-linter (4.0.0)

[warning] [IncorrectDelimiter] The feat/signed-json-compliance-export key has incorrect delimiter

(IncorrectDelimiter)